fi/development.html

```diff
@@ -0,0 +1,210 @@
+         <a href="http://groups.google.com/group/sympy">postituslistalla</a>. Voit myös
+        lisätä bugiraportin (tai kommentin) tällä <a href="https://github.com/sympy/sympy/issues/">sivulla</a>.
+      </li>
+      <li>Voit blogata kuinka käytät SymPy-ohjelmistoa.</li>
+      <li>Voit avustaa meitä kirjoittamaan dokumentaatiota (lähettämällä meille muutostiedoston).</li>
+      <li>Voit avustaa ihmisiä <a href="http://groups.google.com/group/sympy">postituslistalla</a>.</li>
+      <li>Voit avustaa meitä tarkistamalla muutostiedostoja.</li>
+      <li>Voit lähettää meille muutostiedostoja joissa on bugikorjauksia tai uusia ominaisuuksia.</li>
+    </ul>
+  </p>
+  <p>
+    Paras tapa avustaa lahjoittamalla lähdekoodia tai dokumentaatiota on lähettämällä meille muutostiedoston.
+  </p>
+</div>
+
+<h2 class="title">Kuinka lähettää muutostiedosto</h2>
+<div class="inner">
+  <p class="first">
+    Paras tapa on lähettää meille talletuspyynnön
+    <a href="http://github.com/sympy/sympy">sympy/sympy</a> keskussäilöön.
+    Me arvioimme ja yhdistämme sen.
+  </p>
+
+  <p>
+    Mikäli tarvitset apua gitin, githubin, talletuspyyntöjen, muutostiedostojen, ohjelmoinnin, SymPyn tai
+    minkä tahansa muun asiaan liittyvän kanssa, ole hyvä ja lähetä e-mail <a href="https://groups.google.com/forum/#!forum/sympy">postituslistalle</a>
+    tai kysy <a href="https://gitter.im/sympy/sympy">Gitterissä</a>, autamme mielellämme.
+    Lisäksi voit lukea ohjeet
+    <a href="https://github.com/sympy/sympy/wiki/Development-workflow">
+     kuinka valmistella muutostiedostoja</a> käyttäen gitia ja githubia.
+  </p>
+
+  <p>
+    Githubin talletuspyyntö on meille mieluisin tapa, koska se tekee meille helpoksi
+    arvioida ja yhdistää lähdekoodia. Joten,
+    voit lisäksi kloonata viimeisimmän git-säilön (ks. linkki oikealla), valmistella haaran jossa
+    koodisi sijaitsee, ladata sen jonnekin (esimerkiksi <a
+                                                   href="http://github.com/">github</a>) ja lähettää meille linkin <a
+                                                                                                                      href="https://groups.google.com/forum/#!forum/sympy">sympy</a>
+    postituslistalle, tai voit jopa lähettää meille raakamuutostiedostoja --- mutta se tietää meille työtä
+    koska meidän pitää integroida muutokset.
+  </p>
+
+  <p>
+    SymPy-projektilla on laaja <a href="http://docs.sympy.org/dev/aboutus.html#sympy-development-team">kehitystiimi</a>, lähetä meille korjaustiedosto ja sinun nimesi ilmestyy kehitystiimiimme.
+  </p>
+
+</div>
+<h2 class="title">Sivustojen lähdekoodi</h2>
+<div class="inner">
+  <p class="first">
+    Näiden sivujen lähdekoodi on osoitteessa:
+    <a href="http://github.com/sympy/sympy.github.com">http://github.com/sympy/sympy.github.com</a>
+  </p>
+  Voit lähettää meille parannuksia sisältäviä muutostiedostoja vapaasti.
```

Comment:
It is a bit hard to translate "Feel free". I would like to suggest something like "Älä epäröi".